contributing.md

```diff
@@ -19,7 +19,7 @@ it like it was before), then do it again with your own improvements.
 
 ## How to watch and communicate
 
-- Join {doc}`CodeRefinery chat <chat>`, give comments or submit emoji reactions to
+- Join {ref}`CodeRefinery chat <chat>`, give comments or submit emoji reactions to
```

Comment:
Note that this is a bit redundant and it would have been enough to write "{ref}`chat`".

The longer form is useful if you don't want it to display the heading title but instead a different text. But here the text and heading is the same.

Reply:
There are backticks around "chat" and my comment got mis-formatted.

Reply:
Ah ok, thanks for noting, I did not think about that. I put a text for all of them...

Reply:
Is it ok to leave it as is for now?